src/renderer/services/chain/common.ts

```diff
@@ -1,15 +1,15 @@
+import { THORChain } from '@xchainjs/xchain-util'
 import * as FP from 'fp-ts/lib/function'
 import * as O from 'fp-ts/lib/Option'
 import * as Rx from 'rxjs'
 import * as RxOp from 'rxjs/operators'
 
-import { isCrossChain } from '../../helpers/chainHelper'
 import { selectedPoolChain$ } from '../midgard/common'
 
 const isCrossChainDeposit$: Rx.Observable<boolean> = selectedPoolChain$.pipe(
```

Comment:
Maybe we don't need `isCrossChainDeposit$` anymore. Pls check all code if we can remove it or not